src/server/scripts/Kalimdor/zone_desolace.cpp

```diff
@@ -88,12 +88,20 @@ class npc_aged_dying_ancient_kodo : public CreatureScript
                     DoCast(me, SPELL_KODO_KOMBO_DESPAWN_BUFF, true);
 
                     me->UpdateEntry(NPC_TAMED_KODO);
+                    me->CombatStop();
+                    me->DeleteThreatList();
+                    me->SetSpeed(MOVE_RUN, 0.6f, true);
                     me->GetMotionMaster()->MoveFollow(caster, PET_FOLLOW_DIST, me->GetFollowAngle());
+                    me->setActive(true);
                 }
             }
             else if (spell->Id == SPELL_KODO_KOMBO_GOSSIP)
             {
                 me->SetFlag(UNIT_NPC_FLAGS, UNIT_NPC_FLAG_GOSSIP);
+                me->SetHomePosition(me->GetPositionX(), me->GetPositionY(), me->GetPositionZ(), me->GetOrientation());
+                me->GetMotionMaster()->Clear();
+                me->GetMotionMaster()->MoveIdle();
+                me->setActive(false);
```

Comment:
OK, may I have a ruling decision on this, so I know that I have to remove those 2 lines, in case this is the only issue before the PR will be accepted? :smirk_cat: